app/lists.rb

```diff
@@ -0,0 +1,43 @@
+module Listing
+  extend Discordrb::Commands::CommandContainer
+  
+  command :WingInitilise, description: "Initilise the array. In case of derps." do |event|
+	wing_list = Array.new
+  end
+
+  command :WingList, description: "Lists those who are requesting wing." do |event| #Simple, return the text file's contents. May consider making this a array. Also, admin lock this.
+    
+  end
+  
+  command :WingRemove, description: "Remove a person from the wing queue", min_args: 1, max_args: 1 do |event| #Remove someone who is on the wing list. Defo admin lock.
+	
+  end
+
+  command :WingMe, description: "Need a wing? &WingMe" #Adds the person who used the command to the wing list.
```

Comment:
You'll need to add `:min_args`, `:max_args`, and make it a block (that's the `do |event, arg1, arg2, ...argN|` part)